#! /bin/sh /usr/share/dpatch/dpatch-run
## 07_gcc-4.1.dpatch by tbm@cyrius.com
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: build with gcc-4.1

@DPATCH@
--- a/src/pptout/document/group.h~	2006-03-21 03:19:51.000000000 +0000
+++ src/pptout/document/group.h	2006-03-21 03:20:02.000000000 +0000
@@ -30,7 +30,7 @@
   void set_flow_around(bool);
   bool get_flow_around() const;
   Boundary get_obstacle_boundary() const;
-  BoundaryVect Group::obstacle_list() const;
+  BoundaryVect obstacle_list() const;
   
   void add(Pagent* obj);
   Pagent* ungroup(Pagent* obj);
@@ -55,7 +55,7 @@
   /** Raised on direct changes to the group (i.e. new / ungrouped member) */
   sigc::signal<void, Pagent*> group_changed_signal; 
 private:
-  Boundary Group::get_untransformed_box() const;
+  Boundary get_untransformed_box() const;
   
   ChildVec childs;
 };
--- a/src/pptout/propertiesdialog.h~	2006-03-21 03:24:23.000000000 +0000
+++ src/pptout/propertiesdialog.h	2006-03-21 03:24:28.000000000 +0000
@@ -39,7 +39,7 @@
   // no destructor
   ~PropertiesDialog();
 
-  void PropertiesDialog::show_page_contents(GtkNotebookPage*, guint index);
+  void show_page_contents(GtkNotebookPage*, guint index);
 
   Gtk::Notebook book;
   std::vector<PropBase*> pages;
--- a/src/xml2ps/canvas.hh~	2006-03-21 03:09:36.000000000 +0000
+++ src/xml2ps/canvas.hh	2006-03-21 03:10:43.000000000 +0000
@@ -52,7 +52,7 @@
     HBox hbox(bool span, float margin_left, float margin_right, 
 	      float ascender, float descender);
     
-    virtual void xml2ps::Canvas::addRelObstacle
+    virtual void addRelObstacle
     (const float left, const float bottom, const float right, const float top);
     
     // Check where the current column starts.

